integration_test/tests/exampleSpec.ts

```diff
@@ -1,9 +1,17 @@
 import { TestScope } from 'cavy';
+import { loadSourceConfig, playFor, startPlayerTest } from '../playertesting';
+import { SourceType } from 'bitmovin-player-react-native';
 
 export default (spec: TestScope) => {
-  spec.describe('cavy', () => {
-    spec.it('works', async () => {
-      await new Promise((resolve) => setTimeout(resolve, 1000));
+  spec.describe('player', () => {
+    spec.it('loads source and plays for 5 seconds', async () => {
+      await startPlayerTest({}, async () => {
+        await loadSourceConfig({
+          url: 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
+          type: SourceType.HLS,
+        });
+        await playFor(5);
+      });
     });
   });
 };
```

Comment:
As discussed, the test is not a good example to showcase. Here is a suggestion.. if applied please also update the markdown file with one of these examples.

```suggestion
import { TestScope } from 'cavy';
import {
  callPlayerAndExpectEvents,
  EventSequence,
  EventType,
  expectEvents,
  loadSourceConfig,
  RepeatedEvent,
  startPlayerTest,
} from '../playertesting';
import { SourceType } from 'bitmovin-player-react-native';

export default (spec: TestScope) => {
  spec.describe('calling play when a source is loaded', () => {
    spec.it('emits a Play and Playing event', async () => {
      await startPlayerTest({}, async () => {
        await loadSourceConfig({
          url: 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
          type: SourceType.HLS,
        });
        await callPlayerAndExpectEvents((player) => {
          player.play();
        }, EventSequence(EventType.Play, EventType.Playing));
      });
    });
  });

  spec.describe('playing a source', () => {
    spec.it('emits TimeChanged events', async () => {
      await startPlayerTest({}, async () => {
        await loadSourceConfig({
          url: 'https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8',
          type: SourceType.HLS,
        });
        await callPlayerAndExpectEvents((player) => {
          player.play();
        }, EventSequence(EventType.Play, EventType.Playing));
        await expectEvents(RepeatedEvent(EventType.TimeChanged, 5));
      });
    });
  });
};

```

Reply:
Added your changes in 23323d33367ac71d9b9e3fe280a1220c6b917559

Reply:
also updated CONTRIBUTING.md here 3742d685